tests/mocks/MockInstances.ts

```diff
@@ -147,7 +158,11 @@ function addCommand(command: SkyraCommand) {
 	for (const alias of command.aliases) commands.aliases.set(alias, command);
 }
 
-class Command extends SkyraCommand {}
+class Command extends SkyraCommand {
+	public override messageRun(): unknown {
```

Comment:
Implement this as an arrow function then use `vi.fn()`. Should be `public override messageRun = vi.fn()` I think.